src/Castle.Facilities.Quartz/ReleasingJobListener.cs

```diff
@@ -6,16 +6,16 @@
 namespace Castle.Facilities.Quartz
 {
     /// <summary>
-    /// JobListener that will release Jobs out of the Kernel
+    ///     JobListener that will release Jobs out of the Kernel
```

Comment:
Do we need extra indentation in the XML summary comments?

Reply:
Indendations fixed